edb/schema/referencing.py

```diff
@@ -546,8 +640,9 @@ def _build_alter_cmd_stack(self, schema, context, scls, *, referrer=None):
             else:
                 obj = None
 
-        cmd = delta
+        cmd: Union[sd.DeltaRoot, sd.AlterObject[Any]] = delta
```

Comment:
Just `sd.Command` would suffice, since that's what you've annotated the return type to be.